src/Filters/HttpFilter.php

```diff
@@ -132,15 +134,25 @@ protected function addFiltersToQuery(Builder $builder)
     protected function filtersExact(Builder $query, $value, string $property): Builder
     {
         $property = self::sanitize($property);
-
+    
+        if ($query->getModel()->hasCast($property, ['date', 'datetime', 'immutable_date', 'immutable_datetime']) || $property === $query->getModel()->getCreatedAtColumn() || $property === $query->getModel()->getUpdatedAtColumn()) {
+            if (is_array($value) && (isset($value['start']) || isset($value['end']))) {
+                return $query->whereDate($property, '<=', $value['end'] ?? now()->addYears(50))
+                             ->whereDate($property, '>=', $value['start'] ?? now()->subYears(50));
+            }
+        }
+        if (is_array($value) && (isset($value['start']) || isset($value['end']))) {
```

Comment:
number range should be called min,max and should be used similar approach as above